tests/gdn/test_prefill_state_indices.py

```
@@ -0,0 +1,172 @@
+    )
+    return output, final
+
+
+def _make_pool(init_state, perm, n_pool, pad, dtype, device):
+    """Build a state pool holding init_state[i] at row perm[i].
+
+    ``pad > 0`` gives the pool a padded (non-compact) first-dimension stride to
+    exercise the arbitrary-stride path, while keeping the inner [H, V, K] block
+    contiguous -- mirroring TRT-LLM's mamba cache, which packs conv+ssm per slot.
+    """
+    num_seqs, H, D, _ = init_state.shape
```

Ruff (0.15.21)

[warning] Unpacked variable `num_seqs` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)